### PR TITLE
KAFKA-12341: Ensure consistent versions for javassist

### PR DESCRIPTION
**Pull request by ijuma**
And update to 3.27.0-GA.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: gradle/dependencies.gradle [hunk 1]
```diff
@@ -69,6 +69,7 @@ versions += [
   jackson: "2.10.5",
   jacksonDatabind: "2.10.5.1",
   jacoco: "0.8.5",
+  javassist:"3.27.0-GA",
```
Comment (chia7712) [inline gradle/dependencies.gradle]: (code style) it needs a space :)

**Review by chia7712 — APPROVED** [review]
+1 if QA does not complain

File: gradle/dependencies.gradle (hunk 1 above)
Comment (ijuma) [reply gradle/dependencies.gradle]: Fixed.

**Comment by ijuma** [comment]
Unrelated flaky failures:

> Build / JDK 11 / org.apache.kafka.connect.mirror.integration.MirrorConnectorsIntegrationSSLTest.testReplication()
> Build / JDK 11 / kafka.network.DynamicConnectionQuotaTest.testDynamicListenerConnectionCreationRateQuota()
> Build / JDK 8 / kafka.api.TransactionsBounceTest.testWithGroupMetadata()

**Comment by ijuma** [comment]
Merged to trunk and 2.8.0.

**Pull request merged**